### PR TITLE
WIP: first version of the docker file + simple improvements

### PR DESCRIPTION
**Pull request by pvanderlinden**
Just raising this as a WIP for visibility ;)

### CODE REVIEW

File: step1/webconsole.py [hunk 1]
```diff
@@ -16,7 +16,8 @@ def run(self, code):
         output = io.StringIO()
         with contextlib.redirect_stdout(output):
             with contextlib.redirect_stderr(output):
-                self.console.push(code)
+                for line in code.splitlines():
```
Comment (alixedi) [inline step1/webconsole.py:19]: Thank you!

File: step2/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,18 @@
+FROM python:3.7
+
+ADD step1/requirements.txt /tmp
+RUN pip install -r /tmp/requirements.txt
+
+ADD step1/webconsole.py /
+
+# Add Tini
```
Comment (alixedi) [inline step2/Dockerfile:8]: We can ask participants and use Docker CE which has [tini built in](https://github.com/krallin/tini#using-tini)?

**Review by alixedi — APPROVED** [review]
Couple of minor points but LGTM!

File: step2/Dockerfile (hunk 2 above)
Comment (pvanderlinden) [reply step2/Dockerfile:8]: NOTE: If you are using Docker 1.13 or greater, Tini is included in Docker itself. This includes all versions of Docker CE. To enable Tini, just pass the --init flag to docker run

I don't think we have access to that flag from minikube

Comment (alixedi) [reply step2/Dockerfile:8]: Cool. NP.

**Pull request merged**